django/contrib/admin/static/admin/css/base.css

```diff
@@ -20,10 +20,15 @@ a:link, a:visited {
     text-decoration: none;
 }
 
-a:hover {
+a:focus, a:hover {
     color: #036;
 }
 
+a:focus {
+    outline: none;
```

Comment:
Thanks for the feedback guys. My originally thinking was not to rely on the browser and ensure the tabbed item is identifiable by explicitly setting it. Seems that most people agree it's best not to remove the default browser outline so I'll restore it where it was active. Some elements currently do not have any indication of being the focused item, even with the browser outline active, so I think it's best to keep those changes.